packages/replay/src/integration.ts

```diff
@@ -226,6 +226,19 @@ Sentry.init({ replaysOnErrorSampleRate: ${errorSampleRate} })`,
     return this._replay.flushImmediate();
   }
 
+  /**
+   * If not in "session" recording mode, flush event buffer (i.e. creates a new replay).
+   * Unless `continueRecording` is false, the replay will continue to record and
+   * behave as a "session"-based replay.
+   */
+  public async capture(continueRecording: boolean = true): Promise<void> {
```

Comment:
l: We can omit the `: boolean` part here, as that is inferred.

```suggestion
  public async capture(continueRecording = true): Promise<void> {
```

Reply:
l: Other thought: Not sure, but maybe we make this an option object? Just thinking about possible future changes to this API, where it is always easier to add new options if we don't have a bunch of boolean arguments. E.g.

```ts
public async capture(options?: { continueRecording?: boolean }): Promise<void>
```